gslib/utils/json_six.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python
+#
+# Copyright 2019 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""JSON support for python 2 and 3.
+
+Public classes:
+  MessageJSONEncoder: JSON encoder for message objects.
+
+Public functions:
+  encode_message: Encodes a message in to a JSON string.
+  decode_message: Merge from a JSON string in to a message.
+"""
+
+from __future__ import absolute_import
+from __future__ import print_function
+from __future__ import division
+from __future__ import unicode_literals
+
+import json
+from six import PY3
+from six import binary_type
+
+# used for including both loads and dumps methods for ease of importing
+from json import dumps
+
+def loads(json_data):
+  """
```

Comment:
This docstring isn't consistent with google py style.  Doc strings should look like:

```
def foo():
  """Summary line less than 80 chars.

  Additional text if necessary.
  """
```

Reply:
Docstring formatting changed changed and expanded to include more information